### DISCUSSION TITLE
Not able to add navigation. Got error

### DISCUSSION THREAD

**Discussion by rinkiqa12** [statamic/cms#4694] [General]
Hello,

I'm adding or creating navigation using a click on a **link to entry** dropdown option. But I go VUE.js error which **TypeError: this.$refs.input.$refs.existing is undefined**.Could you please help me with how to add or create a navigation or use this feature?
Please see attached screenshot for more clarification. 

Please see screenShot for more clarification.

https://drive.google.com/file/d/12ONWk_vZQhvtJ5MOrb4-Et2AbCYN_p_K/view?usp=sharing

Currently, i'm using **"statamic/cms": "3.2.21"** with **"statamic/eloquent-driver": "^0.1.1"** ,**"edalzell/blade-directives": "3.5.1",** in laravel 

Please composer.json to install all packages.

 "require": {
        "php": "^7.3|^8.0",
        "edalzell/blade-directives": "3.5.1",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "laravel/framework": "^8.54",
        "laravel/sanctum": "^2.11",
        "laravel/tinker": "^2.5",
        "league/commonmark": "^1.5",
        "statamic/cms": "3.2.21",
        "statamic/eloquent-driver": "^0.1.1",
        "vinkla/hashids": "^9.1"
    },

**Comment by jasonvarga** [comment]
What browser?

**Comment by rinkiqa12** [comment]
Thanks for your reply.

Above error on **Firefox**...

And also checked on **chrome** browser and go the below error.

vue.common.prod.js:6 TypeError: Cannot read properties of undefined (reading 'click')
    at a.linkExistingItem (PageSelector.vue:76)
    at a.linkEntries (View.vue:274)
    at click (View.vue?dd60:1)
    at $e (vue.common.prod.js:6)
    at a.n (vue.common.prod.js:6)
    at $e (vue.common.prod.js:6)
    at a.e.$emit (vue.common.prod.js:6)
    at a.selectAndClose (DropdownItem.vue:33)
    at $e (vue.common.prod.js:6)
    at HTMLAnchorElement.n (vue.common.prod.js:6)

Please let me know what should we do?

**Reply by edalzell** [reply]
are you able to create a repo that demonstrates this issue?

**Comment by rinkiqa12** [comment]
No, I am not able to create a repo. Can you help me from my side?
